.changeset/strong-cups-cry.md

```diff
@@ -0,0 +1,38 @@
+---
+"@paprika/list-box": major
+---
+
+💥 The breaking change is:
```

Comment:
I like all the detailed explaination, but this will translate and be put into the `Changelog.md` I'm not sure we wan't this much text inside there.

Thoughts?

Reply:
I agree with Oscar, this might be too long to read in `CHANGELOG.md`, maybe can refer the explanation part from a storybook doc link

Reply:
I would disagree. When I want to update the component `CHANGELOG.md` is the first and the main place I go to get the instructions. I wouldn't afraid to have some extra text in here in case of the breaking change. 

The only thing I'd do differently is slightly change the headers formatting and standardize it. E.g.

instead of 

```
💥 The breaking change is:
```
I'd prefer
```
## 💥 BREAKING CHANGE 
```

Instead of 
```
👷 How a consumer should update their code:
```

```
### 👷 Migrating from 4.x.x to 5.0.1 :
```
for instance.

Reply:
I would also make the long and detailed description only for major updates and if necessary. Otherwise, I'd keep the description short and technical. We also can reference specific PRs if they have a more detailed explanation of the problem that was solved or an improvement that was made.

Reply:
🤷 no idea what others think lol... cc @allison-c would this be problematic? should we stick to a certain amount of details?

Reply:
I will update it with your feedback thank you!

Reply:
there's probably a point where a detailed migration guide becomes too long for a changelog, unless there's a way to do table of contents at the top. I'd be fine with linking out to the full version located elsewhere, as long as the link is prominent in the changelog itself.